database/youtube/machine-learning.json

```diff
@@ -62,5 +62,12 @@
         "category": "youtube",
         "subcategory": "machine-learning",
         "language": "english"
+    },{
```

Comment:
please add brackets in separate lines

Reply:
@rupali-codes  added brackets to separate line

Reply:
thanks